### DISCUSSION TITLE
Daily Standup (Thursday, June 9th)

### DISCUSSION THREAD

**Discussion by antonieto** [CarlosCRG19/ijalti-peers#120] [General]
Share your daily standup update with the team here. Respond to this
discussion before the standup starts in the following format:
```
**What did you achieve in the last 24 hours?**:
 - 

**What are your priorities for the next 24 hours?**:
 - 

**Blockers**:
 -

**Shoutouts**:
 - @username for x 
```

**Comment by antonieto** [comment]
**What did you achieve in the last 24 hours?**:
 - 

**What are your priorities for the next 24 hours?**:
 - Work on SearchBar for aspirants

**Blockers**:
 - Nothing lets gooo

**Shoutouts**:
 - @BrandonMagana for the pair programming session yesterday, @CristianCazares and @DavBot02 for their PR

**Comment by BrandonMagana** [comment]
**What did you achieve in the last 24 hours?**:
 -  Add interested in feature and implemented @DavBot02's requested changes for get aspirant endpoint

**What are your priorities for the next 24 hours?**:
 -  don't know xd

**Blockers**:
 - None

**Shoutouts**:
 - @DavBot02  and @CristianCazares for their awesome job on those 2 PRs, and @antonieto  for yesterdays' PP sesion
